docs/upgrade-to-1.0.md

```diff
@@ -42,6 +42,19 @@ _*backups created prior to v0.10 stored snapshot metadata in the `status.volumeB
     velero backup-location get
     ```
 
+1. Fetch your bucket credentials and make them accessible to Velero
+
+    ```bash
+    # AWS
+    AWS_SHARED_CREDENTIALS_FILE=$(kubectl get secrets/<YOUR_SECRET_NAME> -n velero -o jsonpath="{.data.cloud}" | base64 -d)
```

Comment:
Looks like the `-d` flag is `-D` on macOS; dunno which one we want to default to